force-app/main/default/triggers/Account_Trigger.trigger

```diff
@@ -0,0 +1,13 @@
+trigger Account_Trigger on Account (before insert, after insert) {
+    
+   if(Trigger.isBefore) {
+    AccountTriggerHandler.updateAccountFileds(Trigger.new);  
+
+   }else {
+    if(Trigger.isAfter) {
+        AccountTriggerHandler.afterAccount(Trigger.new);
+
+    }
+  }    
```

Comment:
For practice try switching this around to using the switch statement and trigger operators